assets/images/logo/favicon/site.webmanifest

```diff
@@ -0,0 +1 @@
+{"name":"","short_name":"","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
```

Comment:
Likewise, I don’t see the point here (but that might be coz I don’t know what this is for).

Reply:
This was generated by favicon.io, I think it’s for PWAs, can we leave it in for now?